static/src/components/input/input.js

```diff
@@ -0,0 +1,16 @@
+import tmpl from './input.tml';
+import { renderDOMElement, setEventListener } from '../../utils/DOMApi';
+
+
+export function render({ data, onInput }){
+
+  const handleInput = (evt) => {
+    const input = evt.target;
+    const value = input.value;
+    if (typeof(onInput) === 'function') onInput(value) 
```

Comment:
```suggestion
    if (typeof(onInput) === 'function') {
        onInput(value);
    }
```
Лучше не пропускать скобки в if конструкциях. Без скобок код сложнее читать